modules/gapi/include/opencv2/gapi/fluid/gfluidkernel.hpp

```diff
@@ -49,7 +49,8 @@ class GAPI_EXPORTS GFluidKernel
     enum class Kind
     {
         Filter,
-        Resize
+        Resize,
+        NV12toRGB
```

Comment:
Note the enum is `kind`, not a particular conversion type.
This `Kind` tag affects the way we handle scheduling for kernel (input lines : output lines).

We may support YUV422 (interleaved) for output, and GRGB (Bayer) for input.

Reply:
Interleaved case perfectly fits our recent engine as "line to line" kernel (user himself should interpret his pixel layout in his calculations). Bayer fits our engine too as "two lines to two lines" (2 lpi) kernel (Bayer mosaic with NxN pixels results in NxN rgb image, actually we get 3x number of pixels by interpolating the mosaic). So I propose to keep current name for nv12 kind of in/out lines scheduling as we do not have another precedents